### DISCUSSION TITLE
Go version bump to fix CVE-2023-45288

### DISCUSSION THREAD

**Discussion by AlexXXXVIII** [nginx/nginx-prometheus-exporter#705] [General]
Hi,

we are currently facing CVE-2023-45288 originating from golang version 1.21.5 which is used in the latest nginx-prometheus-exporter version 1.1.0. Is it foreseeable if there will be a new release with a golang version bump in the near future?

Thanks in advance and best regards!

**Comment by lucacome** [comment]
Hi @AlexXXXVIII 

did https://github.com/nginxinc/nginx-prometheus-exporter/releases/tag/v1.2.0 fixed your issue?
